tools/mtouch/MtouchErrors.Designer.cs

```diff
@@ -0,0 +1,2102 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace mtouch {
+    using System;
+    using System.Reflection;
+    
+    
+    [System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "4.0.0.0")]
+    [System.Diagnostics.DebuggerNonUserCodeAttribute()]
+    [System.Runtime.CompilerServices.CompilerGeneratedAttribute()]
+    internal class mtouchErrors {
```

Comment:
I think it would be better to choose a namespace and class name that can be shared between mtouch and mmp (this makes it easier to reference an error message from shared code):

```csharp
namespace Xamarin.Bundler {
    internal class Errors {
    }
}
```

Reply:
I considered this but used separate namespaces/class names because I thought it would be better to separate mtouch and mmp errors. The localized mtouch and mmp errors can already be referenced in shared code, they're separated just under separate namespaces/classes/resx files.

Reply:
You can use #if to separate if you want, but having to #if two different namespaces is rather unpleasant.

Reply:
Using #if isn't necessary. Shared errors can be directly referenced between mtouch and mmp the way both projects are currently set up.

Reply:
Not sure about _confusion_ but I agree it would be easier for _us_ :)

Now we're doing this for our users... so a single file means error duplication or code-overloads is much harder to get thru and confuse our beloved developers.

it also simplify the translator works which in turns could become confusion sources if one translation (for mtouch) differs from another (for mmp).